primevue-gen/build.js

```diff
@@ -0,0 +1,29 @@
+// https://esbuild.github.io/plugins/#using-plugins
+
+const esbuild = require("esbuild");
+
+const myPlugin = {
+  name: "my-plugin",
+  setup(build) {
+    build.onResolve({ filter: /^vue$/ }, (args) => ({
+      path: args.path,
+      namespace: "my-plugin",
+    }));
+
+    build.onLoad({ filter: /.*/, namespace: "my-plugin" }, () => ({
+      contents: "module.exports = Vue",
+    }));
+  },
+};
+(async () => {
+  await esbuild.build({
+    entryPoints: ["./src/main.js"],
+    bundle: true,
```

Comment:
I think we should also add `minify: true,`. It will save some bandwidth and help Promgen load faster.

Non-minified:
```
  ../promgen/static/primevue/main.css  789.7kb
  ../promgen/static/primevue/main.js   311.4kb
```

Minified:
```
  ../promgen/static/primevue/main.css  756.7kb
  ../promgen/static/primevue/main.js   170.6kb
```

At the end of the day, the generated non-minified JavaScript code is already an unreadable mess, so the smaller it is, the better 🤷🏻 .

Reply:
addressed this comment in [8f92674](https://github.com/line/promgen/pull/456/commits/8f926746528ac8ad48958a64a23ba8f0bce8e762)